satpy/multiscene.py

```diff
@@ -45,6 +45,16 @@
 log = logging.getLogger(__name__)
 
 
+def weighted(datasets, weights=None):
```

Comment:
`weighted_stack`? Weighted what?

Reply:
Or maybe we only have one `stack` and weights is an optional kwarg?

Reply:
@djhoese I have now only one function `stack` accessible for the user. In the tests I show how to use it in `test_blend_two_scenes_using_stack_weighted` for instance. 
Right now it calls a private function `_stack_weighted` if a sequence of weights are provided. I kept the two functions separate as the logic is more simple when no weighting is needed, and then the "old" code in stack is much easier to follow.